build/benchmarks.matrix.azure.yml

```diff
@@ -56,4 +56,15 @@ groups:
       template: baselines-scenarios.yml
       profiles:
       - aspnet-azurearm64-lin
+  
+  - jobs:
+      - name: Constant Requests Azure Arm64 Linux
+        template: rate-limited-scenarios.yml
+        profiles:
+        - aspnet-azurearm64-lin
```

Comment:
What about using the current containers-scenarios file and also run the other frameworks with the same frequency. This way all the combinations will work whatever machine is selected in the dashboard, and less templates to maintain.

Reply:
I wasn't sure if we really cared about running against the actual CPU and memory constrained runs, but I see that we do run an unlimited scenario, so we can just ignore the other results.